packages/errors/src/index.ts

```diff
@@ -0,0 +1,8 @@
+export { createErrorClass } from './createErrorClass'
+export { default as ErrorWithCause } from './ErrorWithCause'
```

Comment:
Do we need to have `ErrorWithCause` if we already have `ErrorWithDetails`, which can have both `cause` and `details`?

I'm not clear on the advantage of a cause-only error.